docs/theming.md

```diff
@@ -20,7 +20,7 @@ Available pre-built themes:
 * `adf-pink-bluegrey.css`
 * `adf-purple-green.css`
 
-If you're using Angular CLI you can include one of the prebuilt theme in yours `styles.css` file:
+If you're using Angular CLI you can include one of the prebuilt theme in yours `styles.scss` file:
```

Comment:
Should be "...include one of the prebuilt theme**s** in **your** `style.css` file"